src/sdg/organisaties/views/roles.py

```diff
@@ -29,10 +31,38 @@ class RoleDeleteView(DisallowOwnRoleMixin, RoleBaseMixin, OverheidMixin, DeleteV
     pk_url_kwarg = "role_pk"
     required_roles = ["is_beheerder"]
 
+    def delete(self, request, *args, **kwargs):
+        response = super().delete(request, *args, **kwargs)
+        name = str(self.object).split(" — ")[0]
+        organisation = str(self.object).split(" — ")[-1]
+        messages.success(
+            self.request,
+            _(
+                "De gebruiker {role} van de gemeente {organisatie} is succesvol verwijderd.".format(
+                    organisatie=organisation, role=name
+                )
+            ),
+        )
+        return response
+
 
 class RoleUpdateView(DisallowOwnRoleMixin, RoleBaseMixin, OverheidMixin, UpdateView):
     queryset = Role.objects.all()
     template_name = "organisaties/roles/update.html"
     pk_url_kwarg = "role_pk"
     fields = ["is_beheerder", "is_redacteur"]
     required_roles = ["is_beheerder"]
+
+    def form_valid(self, form):
+        response = super().form_valid(form)
+        name = str(self.object).split(" — ")[0]
+        organisation = str(self.object).split(" — ")[-1]
+        messages.success(
+            self.request,
+            _(
+                "De organisatie role instelling van {role} van de gemeente {organisatie} is succesvol opgeslagen.".format(
```

Comment:
"De gebruikersrol van {role} voor gemeente {organisatie} is succesvol gewijzigd."